packages/grid/src/features/standard-table/util/MultitypeComparator.ts

```diff
@@ -0,0 +1,39 @@
+type ComparableType = number | string | boolean | Date | null | undefined;
+
+export const multitypeComparator = (
+  a: ComparableType,
+  b: ComparableType
+): number => {
+  if (a && b == null) {
+    return -1;
+  }
+  if (a == null && b) {
+    return 1;
+  }
+  if (isTypeOrUndefined(a, b, "number")) {
```

Comment:
why not eliminate the null cases first instead of handling this is all if statements?

Reply:
Oups, that's a mistake.
Null was already being taken care of.